htlcswitch/link.go

```diff
@@ -1942,11 +1944,10 @@ func (l *channelLink) HtlcSatifiesPolicy(payHash [32]byte,
 	// We want to avoid accepting an HTLC which will expire in the near
 	// future, so we'll reject an HTLC if its expiration time is too close
 	// to the current height.
```

Comment:
Is this comment accurate still? The check seems to make sure that we are not creating an outgoing htlc that is close to the broadcast point. This is a stronger check than checking the incoming htlc, because the incoming expiration is always after the outgoing.

Reply:
Good catch.